src/content/blog/content-layer-deep-dive.mdx

```diff
@@ -0,0 +1,242 @@
+
+In Astro 2 we introduced [Content Collections](https://docs.astro.build/en/guides/content-collections/) as a powerful new way to organize your local content, build with type safety, and scale to thousands of pages. Content Collections provide a best-in-class developer experience for local files such as Markdown and MDX, but we heard from you that you wanted the same benefits for all of your content including remote APIs. It was also clear that while many people were building sites with thousands of pages, our Content Collections API struggled to scale into the tens of thousands of pages, with slower builds and excessive memory usage.
+
+In June [we shared an early preview of our plan](https://astro.build/blog/future-of-astro-content-layer/) to solve these problems and more with an all new type of content collection backed by an all new Content Layer API that gives you the flexibility you asked for. We imagined taking collections beyond just files in `src/content`, letting you load your content from anywhere. We have been testing with collections that scale up to a size never before possible. From our first experimental release in Astro 4.14, we have been hard at work stabilizing this new API for release in Astro 5.
+
+## What is the Content Layer
+
+The Content Layer API is the future of the content collections you know and love. It allows you to load data from any source when you build your site, and then access it on your page with a simple, type-safe API. 
+
+The Content Layer API doesn't care where your data is stored. One collection could still be local Markdown files, another could call an API, and yet another could live elsewhere on your filesystem. Using the same `getEntry()` and `getCollection()` functions as before, you can load data from many sources on the same page. There's no performance hit: Astro caches the data locally between builds, meaning that updates can be fast and can minimize the number of API calls that need to be made. And of course, everything is still type-safe, with TypeScript types auto-generated from your schema. 
+
+If you've used content collections before, you'll recognize many of the following concepts and terms. In fact, we haven't changed much about *using* collections in your project at all! A **collection** is still our term for a set of entries that share a common schema. Each entry has a unique ID. It is analogous to a table in a relational database.
+
+But now, each collection uses a **loader**, which defines how the entries are loaded to populate that collection. A loader can be a basic inline function that returns an array of entries, or it can be a more advanced object that handles its own caching and data store. The very first loaders are already distributed as modules on NPM.
```

Comment:
```suggestion
But now, each collection uses a **loader**, which defines how the entries are loaded to populate that collection. A loader can be a basic inline function that returns an array of entries, or it can be a more advanced object that handles its own caching and data store. The very first loaders are already distributed as modules on npm.
```

it's lower-case